Source/ACE/Entity/ModelData.cs

```diff
@@ -11,23 +11,24 @@ namespace ACE.Entity
     /// </summary>
     public class ModelData
     {
-        
-        private List<ModelPallete> modelPalletes = new List<ModelPallete>();
+        public uint PaletteGuid = 0;
```

Comment:
Thanks - I will make the change now and will know that going forward.   I appreciate the feedback.